setup.cfg

```diff
@@ -1,11 +1,11 @@
 [metadata]
-name = pbench
+name = pbench-agent
```

Comment:
Is this file agent-specific? Is it not used for the server?

Reply:
Fixed.

Reply:
What about the analogous line in `project.toml`?